tests/infrastructure/downward-metrics.go

```diff
@@ -93,4 +97,42 @@ var _ = DescribeInfra("downwardMetrics", func() {
 		Expect(found).To(BeTrue())
 		Expect(metrics.Metrics[j].Value).To(Equal("1"))
 	})
+
+	DescribeTable("should not start a VMI if the feature is disabled", func(via libvmi.Option) {
+		By("disabling downwardMetrics feature")
+		tests.DisableDownwardMetrics(virtClient)
+
+		vmi := libvmifact.NewFedora(via)
+		vmi, err := kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		By("checking the phase of the VMI")
+		Consistently(matcher.ThisVMI(vmi), 60*time.Second, 5*time.Second).Should(matcher.BeInPhase(v1.Pending))
```

Comment:
I'd check for the condition instead of waiting 60 secs

Reply:
Adjusted. PTAL!